homewor6/task.js

```diff
@@ -0,0 +1,20 @@
+function quadraticEquation(a, b, c) {
+  try {
+    if (!c) throw "missing parameter";
+    if (a === 0) throw "not a quadratic equation";
+  } catch (err) {
+    throw new ReferenceError(err);
+  }
+
+  const x1 =
+    (-1 * b + Math.sqrt(Math.abs(Math.pow(b, 2) - 4 * a * c))) / (2 * a);
```

Comment:
x1 and x2 should be inside try. 
And wrong formula for calculation x1 and x2. No need abs